### ISSUE TITLE
Docker Permissions

### ISSUE THREAD

**Issue by cc-a** [ImperialCollegeLondon/2020-07-13-Containers-Online#1]
The setup instructions don't make it explicit that users must add their user to the docker group in order to run without sudo. Docker commands in the episodes seem to assume this. Suggest either adding this to the setup or some mention of the need to use sudo.

**Comment by aturner-epcc** [comment]
Thanks @cc-a Is this for Linux? I certainly did not have to do this when I installed Docker Desktop for macOS and the attendees using Windows in the Feb run did not need to do anything like this either.

**Comment by cc-a** [comment]
Yep, a Linux thing. It is mentioned in the linked setup instructions but its quite buried and described as optional. So probably worth making clear it needs to be done if they just want to type in the commands as is.
